config/environments/development.rb

```diff
@@ -42,4 +41,27 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Enable/disable caching. By default caching is disabled.
+  if Rails.root.join('tmp/caching-dev.txt').exist?
```

Comment:
If this isn't an auto-upgrade I wonder if it's redundant with https://github.com/colinxfleming/dcaf_case_management/blob/master/app/controllers/application_controller.rb#L23 ?